product_multi_ean/__init__.py

```diff
@@ -1,22 +1,5 @@
 # -*- coding: utf-8 -*-
```

Comment:
Can you add a README File following https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst?